backend/project-management-system/Project-Management-System-1.0.0.yaml

```diff
@@ -1461,7 +1461,10 @@ definitions:
         minLength: 2
         maxLength: 256
       locality:
-        $ref: 'https://raw.githubusercontent.com/egovernments/egov-services/master/docs/egov-location/contracts/v11-0-0.yml#/definitions/Boundary'
+        type: string
```

Comment:
Are we changing the API spec @chetankolte1

Reply:
Locality was Boundary Object before, now we've MDMS data for locality and the api is expecting locality code which will be string, so I've made the corresponding change in API contract as well